dasharo-compatibility/usb-hid-and-msc-support.robot

```diff
@@ -57,14 +57,16 @@ USB001.003 USB devices detected by OS (Windows)
     Skip If    not ${TESTS_IN_WINDOWS_SUPPORT}    USB001.003 not supported
     Power On
     Login To Windows
-    ${out}=    Execute Command In Terminal    Get-PnpDevice -PresentOnly | Where-Object { $_.InstanceId -match '^USB' }
-    Should Contain    ${out}    HIDClass
-    Should Contain    ${out}    DiskDrive
+    ${out}=    Execute Command In Terminal
+    ...    Get-PnpDevice -PresentOnly | Where-Object { $_.InstanceId -match '^USB' }
+    IF    ${HAS_KEYBOARD}    Should Contain    ${out}    HIDClass
+    IF    ${HAS_USB_STORAGE}    Should Contain    ${out}    DiskDrive
 
 USB002.001 USB keyboard detected in FW
```

Comment:
This still doesn't give reliable results for non-PiKVM platforms.

Reply:
This was/is addressed by @PLangowski in https://github.com/Dasharo/open-source-firmware-validation/issues/206 and https://github.com/Dasharo/open-source-firmware-validation/pull/310